xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -205,8 +205,10 @@ def safe_exec(
                 )
                 log.info(f"Local execution in darklaunch mode produces: {globals_dict} or exception: {emsg}")
             except Exception as e:  # pylint: disable=broad-except
+                # pragma: no cover
```

Comment:
Thanks. This might need to move to the `except` line to cover all this code. See https://github.com/openedx/edx-platform/blob/3401d09fa4298451a65d91ee5fe83026bc4df6de/lms/djangoapps/courseware/testutils.py#L78 for an example.